iron-selectable.html

```diff
@@ -12,50 +12,110 @@
+
+    properties: {
+
+      /**
+       * If you want to use the attribute value of an element for `selected` instead of the index,
+       * set this to the name of the attribute.
+       *
+       * @attribute attrForSelected
+       * @type String
+       */
+      attrForSelected: {
+        type: String,
+        value: null
       },
 
-   Also, the element must manually call the `selectableAttached` from the element's
-   `attached` callback and `selectableDetached` from the element's `detached` callback.
-   callback.
+      /**
+       * Gets or sets the selected element. The default is to use the index of the item.
+       *
+       * @attribute selected
+       * @type String
+       */
+      selected: {
+        type: String,
+        notify: true
+      },
 
-  @group Polymer Mixins
-  @element iron-selectable
-  @homepage github.io
-  */
-  Polymer.Core.Selectable = {
+      /**
+       * Returns the currently selected item.
+       *
+       * @attribute selectedItem
+       * @type Object
+       */
+      selectedItem: {
+        type: Object,
+        readOnly: true,
+        notify: true
+      },
 
-    /* Gets or sets the selected item. The default is to use the index of the item */
-    /* selected */
+      /**
+       * The event that fires from items when they are selected. Selectable
+       * will listen for this event from items and update the selection state.
+       * Set to empty string to listen to no events.
+       *
+       * @attribute activateEvent
+       * @type String
+       * @default 'click'
+       */
+      activateEvent: {
+        type: String,
+        value: 'click',
+        observer: '_activateEventChanged'
+      },
 
-    /* The item that is selected */
-    /* selectedItem */
+      /**
+       * This is a CSS selector sting.  If this is set, only items that matches the CSS selector
+       * are selectable.
+       *
+       * @attribute selectable
+       * @type String
+       */
+      selectable: String,
+
+      /**
+       * The class to set on elements when selected.
+       *
+       * @attribute selectedClass
+       * @type String
+       */
+      selectedClass: {
+        type: String,
+        value: 'iron-selected'
+      },
 
-    /* If you want to use the attribute value of an element for `selected` instead of
-    using the index, set `attrForSelected` to the name of the attribute. */
-    /* attrForSelected: 'name' */
+      /**
+       * The attribute to set on elements when selected.
+       *
+       * @attribute selectedAttribute
+       * @type String
+       */
+      selectedAttribute: {
+        type: String,
+        value: null
+      }
 
-    /* The class to set on elements when selected */
-    selectedClass: 'iron-selected',
+    },
 
-    /* The attribute to set on elements when selected */
-    /* selectedAttribute: 'active', */
+    observers: [
+      '_updateSelected(attrForSelected, selected)'
+    ],
 
     excludedLocalNames: {
       'template': 1
```

Comment:
I filed #4 to track, but merging the PR to use this for menus since this is not blocking.